Description: Removed actdiag from Nova doc as it's currently failing
 Patch like this until something better can be found.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2023-08-09

--- nova-27.0.0.orig/doc/source/reference/scheduling.rst
+++ nova-27.0.0/doc/source/reference/scheduling.rst
@@ -29,35 +29,6 @@ The scheduling process is described belo
     alternative hosts passed between the scheduler and conductor(s) is future
     work.
 
-.. actdiag::
-
-    actdiag {
-        build-spec -> send-spec -> send-reqs -> query -> return-rps ->
-            create -> filter -> claim -> return-hosts -> send-hosts;
-
-        lane conductor {
-            label = "Conductor";
-            build-spec [label = "Build request spec object", height = 38];
-            send-spec [label = "Submit request spec to scheduler", height = 38];
-            send-hosts [label = "Submit list of suitable hosts to target cell", height = 51];
-        }
-
-        lane scheduler {
-            label = "Scheduler";
-            send-reqs [label = "Submit resource requirements to placement", height = 64];
-            create [label = "Create a HostState object for each RP returned from Placement", height = 64];
-            filter [label = "Filter and weigh results", height = 38];
-            return-hosts [label = "Return a list of selected host & alternates, along with their allocations, to the conductor", height = 89];
-        }
-
-        lane placement {
-            label = "Placement";
-            query [label = "Query to determine the RPs representing compute nodes to satisfy requirements", height = 64];
-            return-rps [label = "Return list of resource providers and their corresponding allocations to scheduler", height = 89];
-            claim [label = "Create allocations against selected compute node", height = 64];
-        }
-    }
-
 As the above diagram illustrates, scheduling works like so:
 
 #. Scheduler gets a request spec from the "super conductor", containing
